sdk/extensions/Microsoft.Extensions.Azure/src/AzureClientServiceCollectionExtensions.cs

```diff
@@ -29,7 +29,7 @@ public static void AddAzureClients(this IServiceCollection collection, Action<Az
 
         /// <summary>
         /// Adds the minimum essential Azure SDK interop services like <see cref="AzureEventSourceLogForwarder"/> and <see cref="AzureComponentFactory"/> to the specified <see cref="IServiceCollection"/> without registering any client types.
-        /// Azure SDK log forwarding to to <see cref="ILogger"/> will not be enabled by default, but can be enabled by calling the <see cref="AzureEventSourceLogForwarder.Start"/> method.
+        /// Azure SDK log forwarding to <see cref="ILogger"/> is disabled by default. It can be enabled by calling the <see cref="AzureEventSourceLogForwarder.Start"/> method.
```

Comment:
Should we consider including an `<example>` snippet here to demonstrate how to get at the forwarder to start it?

Reply:
I think that example snippet would feel out-of-place, since it's not a snippet showing how to call the `AddAzureClientsCore` overload. I'll add an example of this when I address https://github.com/Azure/azure-sdk-for-net/pull/30409.